src/library/Constants.cs

```diff
@@ -10,6 +10,7 @@ internal static class Constants
         internal static class AzureDevSpacesService
         {
             public const string UserClusterNamespaceName = "azds";
+            public const string ServiceAnnotations = "bridgetokubernetes/ignore-ports";

```

Comment:
nit: i would make its own class of DeploymentConfig, something like:
internal static class DeploymentConfig
{
       public const string ServiceAnnotations = "bridgetokubernetes/ignore-ports";
}